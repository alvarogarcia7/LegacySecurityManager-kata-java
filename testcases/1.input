root
Root User
12345678
12345678
