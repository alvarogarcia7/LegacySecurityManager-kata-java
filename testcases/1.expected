Enter a username
Enter your full name
Enter your password
Re-enter your password
Saving Details for User (root, Root User, 87654321)

